src/EFCore.PG/Extensions/DbFunctionsExtensions/NpgsqlJsonDbFunctionsExtensions.cs

```diff
@@ -108,4 +108,22 @@ public static bool JsonExistAll(this DbFunctions _, object json, params string[]
     /// </remarks>
     public static string JsonTypeof(this DbFunctions _, object json)
         => throw new InvalidOperationException(CoreStrings.FunctionOnClient(nameof(JsonTypeof)));
+
+    /// <summary>
+    /// Returns true if the JSON value matches the JSON Path expression
+    /// </summary>
+    /// <param name="_">DbFunctions instance</param>
+    /// <param name="json">
+    /// A JSON column or value. Can be a <see cref="JsonDocument"/>, a string, or a user POCO mapped to JSON.
+    /// </param>
+    /// <param name="jsonPath">
+    /// JSON Path query, if the test is true then the function will return true for the given JSON value
+    /// </param>
+    /// <remarks>
+    /// This operation is only supported with PostgreSQL <c>jsonb</c>, not <c>json</c>.
+    /// 
+    /// See https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-SQLJSON-PATH
+    /// </remarks>
+    public static bool JsonPathExists(this DbFunctions _, object json, string jsonPath)
```

Comment:
PG jsonb_path_exists also supports two additional optional parameters (vars, silent) - can you please add support for these?

Reply:
done. Those 2 parameters are optional, but you can't have method parameters with default values in expression trees, so I made overloads. The xml docs didn't really like that though, even with inherit doc , so I had to disable some warnings to make it happy, let me know if you want it fixed another way.